### PR TITLE
Refactor by removing commented out function

### PR DESCRIPTION
**Pull request by Samantha-Stevens**
Removed commented out functions to have cleaner looking code.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -146,11 +148,6 @@ function showOwnPosterForm() {
   mainPoster.classList.add("hidden");
 };
 
-function showSavedPosterPage() {
```
Comment (williamphelps13) [inline src/main.js]: We were placing this function at the bottom of the code

File: src/main.js [hunk 2]
```diff
@@ -160,10 +157,11 @@ function goBackToMainFromSaved() {
   mainPoster.classList.remove("hidden");
   savedPosterPage.classList.add("hidden");
 };
+
 function makePosterDisplayPoster() {
-  var imageUrl = inputPosterImageUrl.value;
```
Comment (williamphelps13) [inline src/main.js]: I think this was as a result of the deletion?

File: src/main.js [hunk 3]
```diff
@@ -174,3 +172,26 @@ function makePosterDisplayPoster() {
       selectQuote.innerHTML = newPoster.quote;
 
 };
+
+function addSavedPoster() {
+  var currentPoster = new Poster(selectImg.src, selectTitle.innerHTML, selectQuote.innerHTML);
+  savedPosters.push(currentPoster);
+};
```
Comment (williamphelps13) [inline src/main.js]: Next step will be to add a If statement to prevent duplicates

**Review by williamphelps13 — APPROVED** [review]

**Comment by joanafbrito** [comment]
We need to fix the conflicts

**Comment by williamphelps13** [comment]
Good practice on merge conflicts! Fix conflicts!

**Pull request merged**